README.md

```diff
@@ -4,7 +4,35 @@ The _JVM Alternative for the Rego Language_ (JARL) is an OPA evaluator for the J
 
 ## Usage
 
-FIXME
+Simple example policy compiled to plan by OPA, and executed by Jarl.
+
+**policy.rego**
+```rego
+package policy
+
+import future.keywords
+
+default allow := false
+
+allow {
+    "admin" in input.user.roles
+}
+```
+```shell
+opa build --target plan --entrypoint policy/allow policy.rego
+```
+This creates a bundle, containing the `plan.json` file. Since this is the only file we care for here, let's extract it
+from the bundle:
+```shell
+tar -zxf bundle.tar.gz /plan.json && rm bundle.tar.gz
+```
+We now have a plan file that we may submit for execution by Jarl!
+```shell
+lein run plan.json '{"user": {"roles": ["admin"]}}'
```

Comment:
Note: this doesn't actually work yet, and we should make sure to fix that ASAP. I'm not sure if it ever did work, but it doesn't now.. and we should have a test to assert that it keeps working as well.

Reply:
There is the beginnings of a `-main` implementation in `src/main/clojure/jarl/core.clj`; but that hasn't been touched for a very long time, and has probably drifted from being serviceable.

Reply:
Yep — I started writing these docs just to have something, but let's fix the implementation tomorrow :)